### ISSUE TITLE
MSBuild Error when using "dotnet build" or "msbuild"

### ISSUE THREAD

**Issue by alishahbaz111** [vescon/MethodBoundaryAspect.Fody#31]
The solution builds fine when using Visual Studio 2017, but fails when using the dotnet build or msbuild command. 
The solution consists of 3 projects: MainProject, MainProject.UnitTests and MainProject.IntegrationTests. 
Building just the main projects succeeds without any errors even when using cli commands, but UnitTests and IntegrationTests fail. Both Tests are referencing the MainProject in Visual Studio.

**Comment by alishahbaz111** [comment]
The error log is pretty much the same as https://github.com/vescon/MethodBoundaryAspect.Fody/issues/20

**Comment by marcells** [comment]
Alright, thanks. I could mark it as duplicate then?

**Comment by marcells** [comment]
In the solution of issue #20 the aspect weaving is failing. There are no unit test projects. It's very simple.

**Comment by alishahbaz111** [comment]
Yes it does. Visual Studio and msbuild on Windows work, but "dotnet build" or "dotnet test" doesn't.
